core/modules/filters/x-listops.js

```diff
@@ -186,4 +204,33 @@ Extended filter operators to manipulate the current list.
         }, []);
         return set;
     };
+
+	/*
```

Comment:
A persistent problem I have -- JsFormat adds an additional indent for the function -- forgot to remove it when I cut and pasted.